parm/config/config.resources

```diff
@@ -196,61 +160,40 @@ elif [ $step = "eupd" ]; then
 
 elif [ $step = "ecen" ]; then
 
-    export wtime_ecen="00:30:00"
+    export wtime_ecen="03:00:00"
```

Comment:
Again, I guess the general comment for a lot of these changes, is why are the wall time requests so much larger, this one is 6x as long. Is that actually what is needed now?

Reply:
changed back to 30 minutes.
Most of  the numbers are copied over from v16rt2, in which some of the larger numbers were set during trouble shootings and testing.  All these numbers need to be revisited before the gfsv16b branches are merged back to develop.

Reply:
v16rt2 wall times are inflated to minimize failures due to long running jobs.    All GFS v16 resource and wall clock limits need to be revisited.   v16rt2 has been and remains a moving target.  It is not a mature, ready-to-implement package.